src/views/Card/Card.js

```diff
@@ -0,0 +1,129 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import {
+  customPropTypes,
+  getUnhandledProps,
+  META,
+  SUI,
+  useKeyOnly,
+} from '../../lib'
+import { createImage } from '../../factories'
+import CardContent from './CardContent'
+import CardDescription from './CardDescription'
+import CardGroup from './CardGroup'
+import CardHeader from './CardHeader'
+import CardMeta from './CardMeta'
+
+function Card(props) {
+  const {
+    centered,
+    children,
+    className,
+    color,
+    description,
+    fluid,
+    header,
+    href,
+    image,
+    meta,
+    onClick,
+    raised,
+  } = props
+
+  const classes = cx('ui',
+    useKeyOnly(centered, 'centered'),
+    useKeyOnly(fluid, 'fluid'),
+    useKeyOnly(raised, 'raised'),
+    color,
+    'card',
+    className,
+  )
+  const rest = getUnhandledProps(Card, props)
+
+  const handleClick = (e) => {
+    if (onClick) onClick(e)
+  }
+  const CardComponent = href || onClick ? 'a' : 'div'
+
+  return (
+    <CardComponent
+      {...rest}
+      className={classes}
+      href={href}
+      onClick={handleClick}
+    >
+      {createImage(image)}
+      {children || <CardContent description={description} header={header} meta={meta} />}
```

Comment:
Content should only be rendered if there is description, header, or meta.  Otherwise, it renders an empty div creating extra whitespace:

<img width=300 src=https://cloud.githubusercontent.com/assets/5067638/17756762/e9145010-6496-11e6-97bd-59f39f363ed6.png>

Also, currently if a user passes children and an image, they'll get a warning but the component will still render an image and their children beside it.  We want to prevent the bad practice altogether.  We've been doing so with explicit if statements and multiple returns.

```jsx
if (children) {
  return (
    <CardComponent {...rest} className={classes} href={href} onClick={handleClick}>
      {children}
    </CardComponent>
  )
}

return (
  <CardComponent {...rest} className={classes} href={href} onClick={handleClick}>
    {createImage(image)}
    {(description || header || meta) && (
      <CardContent description={description} header={header} meta={meta} />
    )}
    {extra && <CardContent extra>{extra}</CardContent>}
  </CardComponent>
)
```

Note that in this ^ code snippet, I've also added support for an `extra` prop on the Card which generates a `<CardContent extra />`.  I've also made it conditional so it doesn't render unless provided.  Let's include this and add it to the propTypes.